styles.css

```diff
@@ -33,7 +33,34 @@ img{
 }
 
 @media(max-width:600px){
-  .main{
-    background:red;
+  main{
+    flex-direction: column;
   }
+  header {
+    order:1;
+  }
+  main {
+    order: 2;
+  }
+  aside{
+    order: 3;
+  }
+}
+footer{
+  order:4;
+}
+}
+
+h1 > header {
+  margin: 0;
+}
+h3, p > footer {
+  margin: 0;
+}
+
+h2 > main {
+  margin: 0;
+}
+h2 > aside {
+  margin: 0;
 }
```

Comment:
When using the direct child syntax (`>`), the parent should go on the left and the child should go on the right.

A great way of writing this if you know that **all** direct children should get the same styling...

```css
header > * {
  /* style goes here */
}
```
> `>` is the direct child selector. `*` means select everything. When put together like in the above example, we are applying styling to all direct children of header.